geometry/meshop.cpp

```diff
@@ -390,6 +502,162 @@ Mesh loadObj(const boost::filesystem::path &filename, ObjMaterial *mtl)
     return parser_.mesh;
 }
 
+/// Check that the following lines contain properties in the given order
+bool hasProperties(std::ifstream& f,
+                   std::string& line,
+                   const std::vector<std::string>& props)
+{
+    for (std::size_t i = 0; i < props.size(); ++i)
+    {
+        char pType[20];
+        char prop[20];
+        sscanf(line.c_str(), "property %s %s", pType, prop);
```

Comment:
Possible buffer overflow. @zap150

Reply:
Rewritten using stringstream, but I am not sure if it's the best style or not.

Reply:
Consult with @vaclavblazek what's the best practice for parsing. It's surely better than the previous solution.